tests-scan

```diff
@@ -107,18 +108,31 @@ def main():
     return 0
 
 
+class Job(NamedTuple):
+    priority: str
+    name: str
+    number: int
+    revision: str
+    ref: str
+    context: str
+    base: str
+    repo: str
+    bots_ref: 'str | None'
+    github_context: str
+
+
 # Prepare a human readable output
-def tests_human(priority, name, number, revision, ref, context, base, repo, bots_ref, github_context, options):
-    if not priority and not options.force:
+def tests_human(job: Job, options: argparse.Namespace) -> str:
+    if not job.priority and not options.force:
         return ""
     return "{name:11} {context:25} {revision:10} {priority:2}{repo}{bots_ref}{branches}".format(
```

Comment:
I'm surprised that you didn't convert to f-string :grin:

Reply:
I was tempted.... :)